src/app/fdctl/run/tiles/fd_rpcserv.c

```diff
@@ -192,6 +193,9 @@ privileged_init( fd_topo_t *      topo,
   args->blockstore = fd_blockstore_join( fd_topo_obj_laddr( topo, blockstore_obj_id ) );
   FD_TEST( args->blockstore!=NULL );
   ctx->blockstore_fd = open( tile->replay.blockstore_file, O_RDONLY );
+  FD_LOG_NOTICE(("%s: %s", tile->replay.blockstore_file, strerror( errno )));
```

Comment:
change this to a `FD_LOG_WARNING` that only runs after doing an if-check (`FD_UNLIKELY`) for an error (-1)